index.html

```diff
@@ -1,327 +1,339 @@
+        <p class="work-card-p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="work-card-ul">
+          <li class="work-card-li">html</li>
+          <li class="work-card-li">bootstrap</li>
+          <li class="work-card-li">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn sub-card-btn" type="button">See Project</button>
+    </div>
+    <!-- Card -->
+    <div class="sub-card">
+      <div class="sub-card-div">
+        <h2 class="work-card-h2">Profesional Art Printing Data</h2>
+        <p class="work-card-p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="work-card-ul">
+          <li class="work-card-li">html</li>
+          <li class="work-card-li">bootstrap</li>
+          <li class="work-card-li">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn sub-card-btn" type="button">See Project</button>
+    </div>
+    <!-- Card -->
+    <div class="sub-card">
+      <div class="sub-card-div">
+        <h2 class="work-card-h2">Profesional Art Printing Data</h2>
+        <p class="work-card-p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="work-card-ul">
+          <li class="work-card-li">html</li>
+          <li class="work-card-li">bootstrap</li>
+          <li class="work-card-li">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn sub-card-btn" type="button">See Project</button>
+    </div>
+    <!-- Card -->
+    <div class="sub-card">
+      <div class="sub-card-div">
+        <h2 class="work-card-h2">Profesional Art Printing Data</h2>
+        <p class="work-card-p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="work-card-ul">
+          <li class="work-card-li">html</li>
+          <li class="work-card-li">bootstrap</li>
+          <li class="work-card-li">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn sub-card-btn" type="button">See Project</button>
+    </div>
+  </div>
+</section>
+
+<!-- About Me -->
+<section class="about-section">
+  <div class="about-div">
+    <h1>About me</h1>
+    <p>
+      Hello I'm a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience! If you
+      like what you see and have a project you need coded, don't hestiate to
+      contact me.
+    </p>
+    <div class="about-btn-div">
+      <button class="btn about-btn" type="button">Get My Resume</button>
+    </div>
+  </div>
+  <div class="about-me-underline">
+    <hr />
+  </div>
```

Comment:
[Optional] I suggest that you consider implementing the horizontal rule to follow that alignments as on the design. It looks quite better.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187044047-37974112-80ed-4de7-88f0-59aec1d899d4.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187044016-be4e402b-0a3c-4da2-8f80-9bcf402cc612.png)


Figma design